src/FBCModelTests.jl

```diff
@@ -18,12 +18,17 @@ include("version.jl")
 include("common.jl")
 include("structs.jl")
 include("frog.jl")
-include("memote.jl")
+include(joinpath("memote", "metabolites.jl"))
+include(joinpath("memote", "reactions.jl"))
+include(joinpath("memote", "basic.jl"))
+include(joinpath("memote", "consistency.jl"))
 
-export frog_generate_report,
-    is_model_charge_balanced,
-    is_model_mass_balanced,
-    has_erroneous_energy_generating_cycles,
-    is_consistent
+# export everything that isn't prefixed with _ (inspired by JuMP.jl, thanks!)
```

Comment:
let's not do this by default; I'll eventuallty add submodules as with cobrexa2.

Reply:
hmm it is basically already in a submodule structure actually. will push latest commit soon